.changeset/famous-queens-dream.md

```diff
@@ -0,0 +1,13 @@
+---
+"@salt-ds/core": minor
+"@salt-ds/lab": minor
+"@salt-ds/theme": minor
+---
+
+Lint rule to strictly only allow for CSS attributes and merged states (`activeDisabled`, `blurSelected`, `partialDisabled`, `selectedDisabled`) to be in camel case within tokens
+
+Deprecated `--size-salt-adornmentGap`; replaced with `--salt-size-adornment-gap`
```

Comment:
This just got [released yesterday](https://github.com/jpmorganchase/salt-ds/releases/tag/%40salt-ds%2Ftheme%401.3.0) and now we're asking the users to change again. 

I think it's time we take a pause and think about how to deal with theme token additional / change. 

@joshwooding @pseys

Reply:
Totally agree, I've already raised this.